packages/app/src/providers/h5grove/h5grove-api.ts

```diff
@@ -35,28 +34,23 @@ import {
 const SUPPORTED_EXPORT_FORMATS = new Set<ExportFormat>(['npy', 'tiff']);
 
 export class H5GroveApi extends DataProviderApi {
-  private readonly fetcher: Fetcher;
-
   /* API compatible with h5grove@2.3.0 */
   public constructor(
     private readonly baseURL: string,
     filepath: string,
-    private readonly axiosConfig?: AxiosRequestConfig,
+    private readonly fetcher: Fetcher = createBasicFetcher(),
     private readonly _getExportURL?: DataProviderApi['getExportURL'],
   ) {
     super(filepath);
-
-    this.fetcher = createAxiosFetcher(
-      axios.create({
-        adapter: 'fetch',
-        ...axiosConfig,
-      }),
-    );
   }
 
   public override async getEntity(path: string): Promise<ProvidedEntity> {
     try {
-      const buffer = await this.fetcher(`${this.baseURL}/meta/`, { path });
+      const buffer = await this.fetcher(`${this.baseURL}/meta/`, {
+        file: this.filepath,
```

Comment:
We no longer expect consumers to configure axios with the `file` parameter, so we have to specify it explicitly. (Note that I had already specified it in the `getValue` and `getAttrValues` methods in my previous PR.)

Reply:
Makes sense